src/types/UiaTextRangeBase.cpp

```diff
@@ -663,7 +663,7 @@ CATCH_RETURN();
 // - pRetVal - the attributeId's sub-type for the first cell in the range (i.e. foreground color)
 // - attr - the text attribute we're checking
 // Return Value:
-// - true, if the attributeId is supported. false, otherwise.
+// - true, if the attributeId is supported. false; otherwise,.

```

Comment:
Oops, this wasn't the right way to correct the rule violation...

```suggestion
// - true, if the attributeId is supported. Otherwise, false.

```